bullet-train.zsh-theme

```diff
@@ -294,10 +318,15 @@ prompt_segment() {
   local bg fg
   [[ -n $1 ]] && bg="%K{$1}" || bg="%k"
   [[ -n $2 ]] && fg="%F{$2}" || fg="%f"
-  if [[ $CURRENT_BG != 'NONE' && $1 != $CURRENT_BG ]]; then
-    echo -n " %{$bg%F{$CURRENT_BG}%}$SEGMENT_SEPARATOR%{$fg%} "
-  else
-    echo -n "%{$bg%}%{$fg%} "
+  if [[ $SEGMENT_PLACE == 'LEFT' ]]; then
+    if [[ $CURRENT_BG != 'NONE' && $1 != $CURRENT_BG ]]; then
+      echo -n " %{$bg%F{$CURRENT_BG}%}$SEGMENT_SEPARATOR%{$fg%} "
+    else
+      echo -n "%{$bg%}%{$fg%} "
+    fi
+  fi
+  if [[ $SEGMENT_PLACE == 'RIGHT' && -n $1 ]]; then
```

Comment:
Are those two cases the only two valid? What would `[ "$SEGMENT_PLACE" = "" ]` mean?
Regardless of above, those two cases (`LEFT`, `RIGHT`) are mutually exclusive, so we can use `else [if]` here. Not for any reasonable performance gain, but for readability.